docs/index.html

```diff
@@ -45,6 +45,7 @@ <h2>Core Modules</h2>
                 <li><a href="makeup-key-emitter/index.html">makeup-key-emitter</a></li>
                 <li><a href="makeup-keyboard-trap/index.html">makeup-keyboard-trap</a></li>
                 <li><a href="makeup-modal/index.html">makeup-modal</a></li>
+                <li><a href="makeup-modal-hoist/index.html">makeup-modal with hoisting</a></li>
```

Comment:
Let's try and stick to the pattern of one demo page per module. Can you just replace the current modal demo page with your new examples (the existing example I had for modal wasn't very good anyway).

Then: 

* Add a checkbox to the demo page that enables/disables hoisting (see https://makeup.github.io/makeup-js/makeup-roving-tabindex/index.html for a similar example of using a checkbox - kind of like a poor man's version of Storybook knobs!)
* Rename buttons from "Host me" to "Modal me"